src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
```

Comment:
The class `FigureSupplier` should not begin with an empty line after the class declaration. Please remove the redundant empty line.